src/loaders/compute_tools/microtrait/Dockerfile

```diff
@@ -14,10 +14,13 @@ RUN micromamba install -y -n $CONDA_ENV -f /tmp/env_microtrait.yaml && \
 
 ENV PATH=$PATH:/opt/conda/bin/
 
+# Install microTrait dependencies
+RUN Rscript -e 'install.packages("kmed", version = "0.4.2", repos = "https://cloud.r-project.org/")' && \
+    Rscript -e 'devtools::install_github("jlw-ecoevo/gRodon")'
```

Comment:
I cannot find a release version for `jlw-ecoevo/gRodon` just master branch unfortunately.